django_redshift_backend/base.py

```diff
@@ -35,9 +37,23 @@ def for_update_sql(self, nowait=False):
         raise NotImplementedError('SELECT FOR UPDATE is not implemented for this database backend')
 
 
+class DatabaseSchemaEditor(BasePGDatabaseSchemaEditor):
+
+    def _model_indexes_sql(self, model):
+        return []
+
+    def _create_like_index_sql(self, model, field):
+        return None
+
+
 class DatabaseWrapper(BasePGDatabaseWrapper):
     vendor = 'redshift'
 
+    SchemaEditorClass = DatabaseSchemaEditor
+    data_types = deepcopy(BasePGDatabaseWrapper.data_types)
+    data_types["AutoField"] = "integer identity(0, 1)"
```

Comment:
Typically auto increment id starts with 1 and my project uses `identity(1, 1)`.
Do you have any special reason to start from 0?